app/javascript/components/CreateSurvey.js

```diff
@@ -9,6 +9,7 @@ import InputLabel from '@mui/material/InputLabel';
 import Box from '@mui/material/Box';
 import Stack from '@mui/material/Stack';
 import Paper from '@mui/material/Paper';
+import DeleteIcon from '@mui/icons-material/Delete';
```

Comment:
Were the changes on this page intentional? You might  be working on an older branch.

Reply:
I think we should remove these file changes

Reply:
Its from 2 weeks ago so the createsurvey js should be removed the rest should be good